docs/source/reference/index.rst

```diff
@@ -16,6 +16,7 @@ Instruments
    :maxdepth: 1
 
    swe
+   hit
```

Comment:
Real nit-picky here but maybe we should set the precedent here for keeping this list in alphabetical order?

Reply:
That's a good point. I'll update that